tests/integration/questionnaires.test.js

```diff
@@ -607,9 +564,9 @@ describe('QUESTIONNAIRES ROUTES - POST /questionnaires/{_id}/card', () => {
   beforeEach(populateDB);
   const payload = { template: SURVEY };
 
-  describe('VENDOR_ADMIN', () => {
+  describe('TRAINING_ORGANISATION_MANAGER', () => {
```

Comment:
Les tests `'should return a 400 if invalid template'` et `'should return a 400 if missing template'` sont redondant, non ?

Reply:
pour moi c'est pas pareil, parce qu'on pourrait très bien se dire que le template n'est pas required mais que si tu le met, tu dois suivre un format. genre les numéros de téléphone, ils doivent suivre un format mais ils ne sont pas toujours obligatoire. Là le test missing template c'est pour s'assurer qu'on ne puisse pas envoyer une requête sans template. On teste le required pas la validité de ce qu'on lui passe